test/add_procs_threading.jl

```diff
@@ -0,0 +1,46 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
```

Comment:
note that this file won't get executed unless you add it to the list in `test/choosetests.jl`